src/parser/avc_parser.cpp

```diff
@@ -1997,44 +2037,82 @@ static inline int CompareLongTermPicNumAsc(const void *p_pic_info_1, const void
     }
 }
 
+static inline int CompareFrameNumWrapDesc(const void *p_pic_info_1, const void *p_pic_info_2) {
+    int frame_num_wrap_1 = ((AvcVideoParser::AvcPicture*)p_pic_info_1)->frame_num_wrap;
```

Comment:
please use c++ style cast (reinterpret_cast) for p_pic_info_1 and p_pic_info_2 on lines 2041-2042 and 2054-2055.